app/models/notification.rb

```diff
@@ -0,0 +1,10 @@
+class Notification
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  belongs_to :user
+  field :message, type: String
+  field :link, type: String
```

Comment:
I won't put it as link, this will probably be a reference to the review/post

Reply:
Also, the message, i think we should define some constants with a message attached to them, probably we will have 2/3 notification types.